starter_code/public/javascript/index.js

```diff
@@ -1,23 +1,52 @@
-const charactersAPI = new APIHandler("http://localhost:8000")
+const charactersAPI = new APIHandler("http://localhost:8000/");
 
-$(document).ready( () => {
-  document.getElementById('fetch-all').onclick = function(){
+$(document).ready(() => {
+  document.getElementById("fetch-all").onclick = function() {
+    event.preventDefault();
+    charactersAPI.getFullList();
+  };
 
-  }
-  
-  document.getElementById('fetch-one').onclick = function(){
-    
-  }
-  
-  document.getElementById('delete-one').onclick = function(){
-        
-  }
-  
-  document.getElementById('edit-character-form').onsubmit = function(){
-            
-  }
-  
-  document.getElementById('new-character-form').onsubmit = function(){
-                
-  }
-})
+  document.getElementById("fetch-one").onclick = function() {
+    event.preventDefault();
+    let id = document.getElementById("character-id").value;
+    console.log(id);
+    charactersAPI.getOneRegister(id);
```

Comment:
You need to consume this promise:
``
charactersAPI.getOneRegister(id).then(character => {
// do stuff on the DOM
});
``